### PR TITLE
컬렉션 상세 페이지 내 스페이스 정보 표시하기

### PR DESCRIPTION
**Pull request by x86chi**

### CODE REVIEW

**Comment by x86chi** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #881** <a href="https://app.graphite.dev/github/pr/penxle/penxle/881?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #882** <a href="https://app.graphite.dev/github/pr/penxle/penxle/882?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/penxle/penxle/882?utm_source=stack-comment).

File: apps/penxle.com/src/routes/(default)/[space]/(space)/collections/[entity]/+page@(default).svelte [hunk 1]
```diff
@@ -119,32 +152,80 @@
...
+          {#if !$query.space.meAsMember}
+            {#if $query.space.followed}
+              <Button
+                class="rounded-12!"
+                color="tertiary"
+                size="md"
+                variant="outlined"
+                on:click={async () => {
+                  await unfollowSpace({ spaceId: $query.space.id });
+                  mixpanel.track('space:unfollow', { spaceId: $query.space.id, via: 'post' });
```
Comment (devunt) [inline apps/penxle.com/src/routes/(default)/[space]/(space)/collections/[entity]/+page@(default).svelte:176]: P1: `via: 'collection'`

File: apps/penxle.com/src/routes/(default)/[space]/(space)/collections/[entity]/+page@(default).svelte [hunk 2]
```diff
@@ -119,32 +152,80 @@
...
+                class="rounded-12!"
+                size="md"
+                on:click={async () => {
+                  if (!$query.me) {
+                    loginRequireOpen = true;
+                    return;
+                  }
+
+                  await followSpace({ spaceId: $query.space.id });
+                  mixpanel.track('space:follow', { spaceId: $query.space.id, via: 'post' });
```
Comment (devunt) [inline apps/penxle.com/src/routes/(default)/[space]/(space)/collections/[entity]/+page@(default).svelte:193]: P1: `via: 'collection'`

**Review by devunt — CHANGES_REQUESTED** [review]

**Review by devunt — APPROVED** [review]

**Comment by devunt** [comment]
### Merge activity

* **Dec 18, 6:11 AM**: [Graphite](https://app.graphite.dev/github/pr/penxle/penxle/882) rebased this pull request after merging its parent, because this pull request is set to merge when ready.

**Pull request merged**